src/plugins/linked.js

```diff
@@ -154,7 +147,6 @@ export default class LinkedPlugins implements IPluginManager {
     if (!this.config.debug) this.out.action.start(`Installing dependencies for ${p}`)
     let yarn = new Yarn(this.out, p)
     await yarn.exec([], {cwd: p})
-    touch(path.join(p, 'node_modules'))
```

Comment:
@dickeyxxx we still need this because nested changes do not bump node_modules and we would keep installing over & over.  For example if I had `node_modules/foo` and the `package.json` bumps `foo` version then it would update, bumping the `node_modules/foo` timestamp but not the `node_modules` timestamp.  This would cause it to install every time after until another dep was added.